components/feature/webnotifications/src/main/java/mozilla/components/feature/webnotifications/WebNotificationFeature.kt

```diff
@@ -70,13 +70,13 @@ class WebNotificationFeature(
             notificationManager?.cancel(NOTIFICATION_TAG_ID, it)
         }
 
-        pendingRequestId++
-        notificationId++
+        val notificationId = NotificationIds.getNextIdForTag(context, NOTIFICATION_TAG_ID)
+        val pendingIntentId = NotificationIds.getNextIdForTag(context, PENDING_INTENT_TAG_ID)
```

Comment:
So we always need a new tag and never want to reuse one?

Reply:
Yes, in this case we need a new one.  Not the old one.